### ISSUE TITLE
Thermostat “Off” gaps get connected

### ISSUE THREAD

**Issue by coneslayer** [beestat/app#119]
When the thermostat is in the Off mode, Beestat stops drawing the dashed orange and blue lines for the setpoints. This behavior is similar to the Ecobee website, and provides a visual indicator that the temperature is not being controlled.

However, when the thermostat is placed back into Auto, the setpoint lines are connected across the Off interval, instead of leaving a gap. It would be more clear what the thermostat was doing if the gap remained to show when the thermostat was off.

![Off but setpoint lines are drawn by coneslayer](https://i.imgur.com/NE1LTNv.jpg)

I have observed this behavior in Chrome (Windows), Firefox (Windows) and Mobile Safari (iOS).

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by coneslayer** [comment]
Poking around a bit more, some "Off" intervals are properly shown as gaps, and others aren't. I'm not sure what the pattern is, but maybe it depends on what's on each side or where it is relative to the edge of the figure. In the screenshot below, where I've zoomed out to the 7-day view, there are large gaps to the left reflecting the Off period, but the same Off period that I highlighted above remains shown with setpoints.

![image](https://user-images.githubusercontent.com/4835005/58287137-4ea07280-7d7e-11e9-969a-af29e9511cdd.png)
